### ISSUE TITLE
[Programmatic access - ADF - Azure data factory] Section 508 502.3.1: (StorageExplorer/document '') The Name property of a focusable element must not be null.

### ISSUE THREAD

**Issue by Shalini3457** [microsoft/AzureStorageExplorer#3838]
**User Experience:**
User will find difficulty to identify the control if that control doesn't have name property as user won't be proceed further.

**Test Environment:**
Application Version: 1.16.0-dev
Build Number: 20200921.1

**Note:** User credentials should NOT be included in the bug.

**Pre-Requisite:**
Navigate to the storage expoler.exe and select it  

**Repro Steps:​**
1. Launch Azure Storage Explorer app.
2. Navigate and select Add data factory control at left panel.
3. Navigate to the various controls like account, azure active directory, subscription etc. on Azure data factory screen.
4. Now, run accessibility insights for Windows.
5. Verify name property of focusable element.

**Actual Result:**
Name property of focusable element is null on Azure data factory window screen.

**Expected Result:**
The Name property of a focusable element must not be null n Azure data factory window screen.    
Application: Storage Explorer has the following accessibility issue that needs investigation.

**Issue Details**
Provide a UI Automation Name property that concisely identifies the element.

**Element path: document ''**

To Reproduce and How to Fix guidance: Please open the attached Accessibility Insights for Windows Test File (*.a11ytest) to reproduce the error in Accessibility Insights for Windows.

This accessibility issue was found using Accessibility Insights for Windows, a tool that helps debug and find accessibility issues earlier.

![MAS4 1 2_Azure data factory](https://user-images.githubusercontent.com/72871089/101026283-bbfca880-359c-11eb-91b3-348d768e682f.png)
